Sources/ClientRuntime/Networking/Http/URLSession/URLSessionHTTPClient.swift

```diff
@@ -502,34 +502,41 @@ public final class URLSessionHTTPClient: HTTPClient {
                     body = .data(nil)
                 }
 
```

Comment:
Values used by telemetry are captured, then a Task is spawned that does the writing of the telemetry data.
This allows for:
- Request can proceed while telemetry is written
- Task provides async context for access to actors.